app/Http/Requests/Three/PostRequest.php

```diff
@@ -30,7 +30,7 @@ public function rules()
             'type' => 'required|string|in:photo,voter-reg,text',
             'action' => 'required|string',
             'why_participated' => 'nullable|string',
-            'text' => 'required|nullable|string|max:140',
+            'text' => 'required|nullable|string|max:255',
```

Comment:
did this not work at 256?

Reply:
oh oops - I thought that `255` would let you do 256 characters - I can change that! But the database doesn't need to be updated, was that it?

Reply:
In the DB we set the column type to be VARCHAR 255. And that means the column should be able to hold 256 character strings. 

So then in the validation we just want to limit it to 256. But it might be worth double checking that a 256 character string is able to be stored in that column which will determine if any DB updates need to be made.